### DISCUSSION TITLE
Report/Display ONLY matches

### DISCUSSION THREAD

**Discussion by dooleyj** [WinMerge/winmerge#1059] [Q&A] [answered]
I have two text files, one is a list of game titles I own, one is a list of game titles available from Steam via GeForece Now cloud gaming.
I want a list of game titles I own that are also on Steam via GeForce Now.
Is there a way to do this?  

I know I can scroll through the basic comparison display and see the matches, but how can one only display the matches or output the matches only?

**Comment by sdottaka** [comment, marked as answer]
Try checking View → Diff Context → 0 lines menu item, and then View → Diff Context → Invert (Hide Different Lines) menu item.
You may need to sort the files before comparing.
![image](https://user-images.githubusercontent.com/98126/143603762-292fea6f-f4d3-4597-a056-5d3f1fbed82b.png)
![image](https://user-images.githubusercontent.com/98126/143603873-5a56f4f0-d6a0-4258-a34a-414591a8e481.png)

**Reply by dooleyj** [reply]
Thanks that did the trick.  I never understood what the whole menu section was about.
